Pipes - <Roxanne Agerone> - <Calculator>

```diff
@@ -0,0 +1,83 @@
+# adds with  add and +
+# subtracts with subtract and -
+# adds with multiply and *
+ # adds numbers with divide and /
+ # handles divide when attempting to divide by 0
+ # handles erroneous input
+
+ def number_question
+   numbers = []
+   puts "give me two numbers"
+       num = gets.chomp.to_f
+       numbers.push(num)
+       num = gets.chomp.to_f
+       numbers.push(num)
+end
+
+ def add(a,b)
+   a + b
+ end
+
+ def subtract(a, b)
+   a - b
+ end
+
+def multiply(a, b)
+  a * b
+end
+
+def divide(a, b)
+  if b == 0
+    puts "Dividing by zero is tricky!  We will discuss this when we do calculus.
```

Comment:
I like that you've split these out into separate methods! It may seem a little pedantic for things like + and -, but for something like division where you have an extra check to make it results in much cleaner code.